### PR TITLE
Travis-ci: updated  support for ppc64le in latest way and added for i…

### PR DESCRIPTION
**Pull request by sanjaymsh**
Hi,
I have added support for ppc64le build on travis-ci in the branch . The travis-ci build log can be tracked on the link :https://travis-ci.com/github/sanjaymsh/dumb-init/builds/188136243 . I believe it is ready for the final review and merge.
Please have a look on it and if everything looks fine for you then please approve it for merge.

Thanks !!

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,14 +8,16 @@ matrix:
         - env: ITEST_TARGET=itest_bionic
         - env: ITEST_TARGET=itest_buster
         - env: ITEST_TARGET=itest_tox
-        - os: linux-ppc64le
+        - arch: ppc64le 
+          env: ITEST_TARGET=itest_tox
```
Comment (asottile) [inline .travis.yml:12]: this isn't needed on ppc64le

**Review by asottile — COMMENTED** [review]
I'm curious what the motivation is -- is the old way going away?

**Comment by sanjaymsh** [comment]
> I'm curious what the motivation is -- is the old way going away?

Yes @asottile , we have already upgraded to new way and using the same. And in case if we are finding the old way anywhere then we are replacing it by new way.

Thanks !!

**Comment by asottile** [comment]
> 
> 
> > I'm curious what the motivation is -- is the old way going away?
> 
> Yes @asottile , we have already upgraded to new way and using the same. And in case if we are finding the old way anywhere then we are replacing it by new way.
> 
> Thanks !!

I'm -0 on this, if the old way works and isn't going away this PR seems mostly just noise?

**Comment by gerrith3** [comment]
@asottile  The folks at Travis have updated the semantics and yaml to make it consistent across all architectures and made per-architecture builds much easier.  The old way is probably going away, not sure when we will decommission those queues, though.  But the goal is to make then use "arch:" universally which you do for everything but ppc64le?  Seems like this patch cleans that up while future-proofing you from the day we turn off that flag.

**Comment by chriskuehl** [comment]
Thanks!

**Pull request merged**